docs/reference/query.rst

```diff
@@ -162,6 +162,20 @@ Dates are written separated by hyphens, like ``year-month-day``, but the month
 and day are optional. If you leave out the day, for example, you will get
 matches for the whole month.
 
+You can also use relative dates to the current time.
+Relative dates begin by the ``@`` character, followed by an optional ``+`` or
+``-`` sign that will increment or decrement now, followed by the time quantity
+that will be represented as an integer followed by either ``d`` for days, ``w``
+for weeks, ``m`` for months and finally ``y`` for year.
+
+Here is an example that finds all the albums added between now and last week::
+
+    $ beet ls -a 'added:@-1w..'
+
+Find all items added in a 2 weeks period 4  weeks ago::
```

Comment:
I hope the change I made satisfies your request, I'm a total newb on .rst format and sadly couldn't find a way in pycharm to get a wysiwyg, is there a way ?